src/ConfigProvider.php

```diff
@@ -27,15 +29,17 @@ public function __invoke(): array
 
     /**
      * Returns the dependency (service manager) configuration
+     *
+     * @return DependencyConfigArray
      */
     public function getDependencyConfig(): array
     {
         return [
             // Legacy Zend Framework aliases
             'aliases'            => [
-                LegacyInjectorInterfae::class  => InjectorInterface::class,
-                LegacyConfigInterface::class   => ConfigInterface::class,
-                LegacyInjectorGenerator::class => CodeGenerator\InjectorGenerator::class,
+                'Zend\Di\InjectorInterface'               => InjectorInterface::class,
+                'Zend\Di\ConfigInterface'                 => ConfigInterface::class,
+                'Zend\Di\CodeGenerator\InjectorGenerator' => CodeGenerator\InjectorGenerator::class,
```

Comment:
Is this still a thing? Could we drop this since this package is now marked as conflicting to `zend-di`?

Reply:
Not really: these string keys were still strings, and not class aliases

Reply:
Do you have a good idea on how to get rid of this gracefully?